libraries/AP_Baro/AP_Baro.cpp

```diff
@@ -981,6 +981,17 @@ void AP_Baro::update(void)
 #endif
 }
 
+#ifdef HAL_BUILD_AP_PERIPH
+// calibration and alt check not valid for AP_Periph
+bool AP_Baro::healthy(uint8_t instance) const {
+    return (instance < BARO_MAX_INSTANCES) ? sensors[instance].healthy : false;
+}
+#else
+bool AP_Baro::healthy(uint8_t instance) const {
+    return (instance < BARO_MAX_INSTANCES) ? (sensors[instance].healthy && sensors[instance].alt_ok && sensors[instance].calibrated) : false;
```

Comment:
an if statement would be a lot clearer here if you don't mind

Reply:
No worries! Done :)